mlflow/server/validation.py

```diff
@@ -0,0 +1,35 @@
+import re
+from typing import List, Union
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+
+VALID_CONTENT_TYPE_CHARS = re.compile(r"^[a-zA-Z0-9\-/;,+_ ]*$")
+
+
+def _validate_content_type(flask_request, allowed_content_types: Union[str, List[str]]):
```

Comment:
```suggestion
def _validate_content_type(flask_request, allowed_content_types: List[str]):
```

Can we only accept a list?

Reply:
Sure, no strong reason for allowing both here.